start.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# Install rsync if it's not already installed
+command -v rsync &> /dev/null || sudo apt-get install -y rsync
+
+# Install Garden if it's not already installed
+command -v garden &> /dev/null || (curl -sSL https://github.com/garden-io/garden/releases/download/0.13.0/garden-0.13.0-linux-amd64.tar.gz | tar xz && \
```

Comment:
Should we do a shortlink to track how many downloads we get because of this link? <3

Reply:
@ShankyJS I put a shortlink button up at the top: users will necessarily click the button, then run the script to get started so I don't think we need to double-tap this.